Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG node_version=12
+ARG crystal_version=1.0.0
+
+FROM node:${node_version}-alpine as frontend-build
+COPY /frontend /src/frontend
+WORKDIR /src/frontend
+COPY package*.json ./
+
+RUN npm install -g @angular/cli @angular-builders/custom-webpack && npm install
+RUN npx ng build --prod
+
+###########################
+
+FROM crystallang/crystal:${crystal_version}-alpine
+WORKDIR /src
```

Comment:
Might be cleaner to have this as `/app` to prevent confusion with the actual `src` directory